pkg/server/plugin/upstreamca/awssecret/awssecret.go

```diff
@@ -42,6 +42,7 @@ type AWSSecretConfiguration struct {
 	AccessKeyID     string `hcl:"access_key_id" json:"access_key_id"`
 	SecretAccessKey string `hcl:"secret_access_key" json:"secret_access_key"`
 	SecurityToken   string `hcl:"secret_token" json:"secret_token"`
+	AssumeRoleARN   string `hcl:"assumerole_arn" json:"assumerole_arn"`
```

Comment:
Perhaps this should be `assume_role_arn`? Trying to get more consistent about naming, case etc